plugins/ethHelper.js

```diff
@@ -0,0 +1,38 @@
+import EthHelper from '@likecoin/likecoin-eth-util';
```

Comment:
Prefer the filename is the same as the package name
`EthHelper` -> `EthUtil`???